docker/Dockerfile-netty-centos6

```diff
@@ -0,0 +1,22 @@
+FROM centos:6
+MAINTAINER netty@googlegroups.com
+ENTRYPOINT /bin/bash
+
+ENV SOURCE_DIR $HOME/source
+ENV MAVEN_VERSION 3.5.2
+ENV JAVA_VERSION 1.8.0
+
+RUN mkdir $SOURCE_DIR
+WORKDIR $SOURCE_DIR
+
+# install dependencies
+RUN yum install -y lsb-core autoconf automake libtool make tar glibc-devel openssl-devel apr-devel lksctp-tools java-$JAVA_VERSION-openjdk-devel git wget
```

Comment:
Consider sorting and multi-lining for easier maintenance: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#sort-multi-line-arguments